src/SUMMARY.md

```diff
@@ -17,7 +17,9 @@
   - [Stateless clients](./use-cases/stateless-clients.md)
 - [Resources](./resources.md)
 - [Development](./development/intro.md)
-  - [Verkle replay](./development/verkle-replay.md)
+  - [Mainnet analysis](./development/mainnet-analysis/intro.md)
```

Comment:
Ended up using `Mainnet analysis` instead of `Mainnet insights`.
No strong preferences from my side.